### PR TITLE
Noting Features of both JIRA and GitHub and their compatability

### LINKED ISSUE #3: Document data models and migration patterns, note complexity of a field to migrate

### PR DESCRIPTION
**Pull request by IanSinColorado**
Adding a description of features that JIRA and GitHub have separately. Listing which ones can migrate directly and indirectly

### CODE REVIEW

File: ticketStructure.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# JIRA Ticket Features:
```
Comment (jwfuller) [inline ticketStructure.md]:
```suggestion
# JIRA Ticket Fields:
```

File: ticketStructure.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+- Attachments
+- Issue Links
+- Comments
+- Assignee
+- Reporter
+- Votes
+- Watchers
+- Dates (created and last modified)
+
+# GitHub Issue Features:
```
Comment (jwfuller) [inline ticketStructure.md]:
```suggestion
# GitHub Issue Fields:
```

**Review by jwfuller — APPROVED** [review]
This looks great!. One minor tweak to wording. Is there anything that looks difficult to map over?

**Comment by IanSinColorado** [comment]
One thing that looks difficult is keeping the JIRA ticket and GitHub issue creator consistent. I think the checklist and attachments can be added to the description.

**Pull request merged**